src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,95 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] storage = (T[]) new Object[DEFAULT_CAPACITY];
```

Comment:
Let's initialize your storage the same way inside the constructor with zero parameters